subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/MultiProjectDependencyIntegrationTest.groovy

```diff
@@ -193,7 +193,7 @@ project(':c') {
         failure.assertHasCause 'failure in b'
 
         and:
-        jarsNotBuilt 'a', 'b', 'c', 'd'
+        jarsNotBuilt 'a', 'b'
```

Comment:
why did this change?

Reply:
It didn't need to - an earlier change caused this test to fail, which got me looking at it, which made me realize the c and d check here is totally unnecessary.  I fixed the issue with the test, but I also removed the check.

It's not germane to this PR - I reverted it (it doesn't really affect things either way).